packages/sdk/test/marketplace.test.ts

```diff
@@ -31,30 +38,31 @@ let tokenAddress = NATIVE_TOKEN_ADDRESS;
  * Bog and Sam and Abby wallets will be used for direct listings and auctions.
  */
 describe("Marketplace Contract", async () => {
-  let marketplaceContract: Marketplace;
-  let dummyNftContract: NFTCollection;
-  let dummyBundleContract: Edition;
-  let customTokenContract: Token;
+  let marketplaceContract: MarketplaceImpl;
+  let dummyNftContract: NFTCollectionImpl;
+  let dummyBundleContract: EditionImpl;
+  let customTokenContract: TokenImpl;
 
   let adminWallet: SignerWithAddress,
     samWallet: SignerWithAddress,
-    abbyWallet: SignerWithAddress,
+    // abbyWallet: SignerWithAddress,
     bobWallet: SignerWithAddress,
-    w1: SignerWithAddress,
-    w2: SignerWithAddress,
-    w3: SignerWithAddress,
+    // w1: SignerWithAddress,
+    // w2: SignerWithAddress,
+    // w3: SignerWithAddress,
```

Comment:
if you insist in cleaning up the test files, remove these and the other commented stuff xD (you asked for it)